packages/safe-tools-client/app/components/setup-safe-modal/index.gts

```diff
@@ -39,44 +54,51 @@ export default class SetupSafeModal extends Component<Signature> {
             <li>You are the owner of the safe. Only the safe owner can schedule
               payments.
             </li>
-            <li>Once you have scheduled payments, the module (source code here)  {{! TODO: Add link }}
+            <li>Once you have scheduled payments, the module
+              <a
+                href='https://github.com/cardstack/cardstack-module-scheduled-payment'
+                target='_blank'
+                rel='external'
+              >
+                (source code here)
+              </a>
               triggers the payments at the appointed time.
             </li>
             <li>Creating the safe and enabling the module requires a one-time gas
-              fee.
+               fee.
             </li>
           </ul>
-          {{! TODO: Gas cost should be handled on this component or be a param ? }}
-          <b>Estimated gas cost: {{gasCost}}</b>
-          {{#if hasEnoughGas}}
-            <div class='safe-setup-modal__section-wallet-info'>
-              {{svgJar
-                'icon-check-circle-ht'
-                class='safe-setup-modal__section-icon'
-              }}
-              <p>
-                Your wallet has sufficient funds to cover the estimated gas cost.
-              </p>
-            </div>
-          {{/if}}
-          {{! TODO: What's the behavior for not having enough gas ? }}
+          <b>Estimated gas cost: {{this.gasCost}}</b>
```

Comment:
Might be helpful to also provide the wallet address that is going to own the safe? Since this lives in a modal, and it can be troublesome for users to confirm what wallet address is connected.

Reply:
Is creation of a new safe is necessary for each chain that the user intends to use? If so, would it also be good to include the network here and to highlight that point?

Reply:
I guess the address will be already visible in the top right corner once we have it connected, but it's ok to add to the modal too. Now the second question I'm not aware of.

Reply:
Ya, address might be visible in the top right, though it will probably have poorer contrast and also be less prioritized visually because of the overlay